tests/test_job.py

```diff
@@ -52,15 +51,6 @@ def builtins_dict():
 NESTED_HASH = "bd6f5828f4410b665bffcec46abeb8f3"
 
 
-def config_from_cfg(cfg):
-    cfile = io.StringIO("\n".join(cfg))
-    return signac.common.config._read_config_file(cfile)
-
-
-def testdata():
-    return str(uuid.uuid4())
-
-
```

Comment:
Yes, this would be better served by [text()](https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.text).  😁